### ISSUE TITLE
Uncaught TypeError: Cannot read property 'editor' of null

### ISSUE THREAD

**Issue by slava-viktorov** [tvooo/scope-inspector#14]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.166.0
**System**: linux 3.17.1-52.g5c4d099-desktop
**Thrown From**: [scope-inspector](https://github.com/tvooo/scope-inspector) package, v0.2.1

### Stack Trace

Uncaught TypeError: Cannot read property 'editor' of null

```
At /home/slava/.atom/packages/scope-inspector/lib/scope-inspector-view.coffee:60

TypeError: Cannot read property 'editor' of null
  at ScopeInspectorView.module.exports.ScopeInspectorView.onClickIdentifier (/home/slava/.atom/packages/scope-inspector/lib/scope-inspector-view.coffee:60:29)
  at HTMLLIElement.<anonymous> (/home/slava/.atom/packages/scope-inspector/lib/scope-inspector-view.coffee:21:41)
  at HTMLLIElement.handler (/usr/local/share/atom/resources/app/src/space-pen-extensions.js:112:34)
  at HTMLDivElement.jQuery.event.dispatch (/usr/local/share/atom/resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at HTMLDivElement.elemData.handle (/usr/local/share/atom/resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)

```

### Commands

```
  3x -0:00.0 core:backspace (input.hidden-input)
```

### Config

```
{
  "core": {
    "disabledPackages": [
      "linter-flow",
      "Remote-FTP",
      "remote-edit",
      "remote-sync",
      "SFTP-deployment",
      "language-javascript-subtlegradient",
      "atom-jshint"
    ]
  },
  "scope-inspector": {
    "userId": "f111d4b4-3d69-437e-a484-4e4f21aee6d1"
  }
}
```

### Installed Packages

```coffee
# User
atom-deconsole, v0.0.1
autocomplete-plus, v0.22.14
bemhtml, v0.10.0
emmet, v2.2.1
gulp-snippets, v0.6.0
javascript-snippets, v1.0.0
js-refactor, v0.3.0
jsformat, v0.7.14
language-css-plus, v0.3.0
language-javascript-semantic, v0.1.0
linter, v0.9.0
linter-csslint, v0.0.11
linter-jscs, v1.4.9
linter-jshint, v0.1.0
refactor, v0.4.1
remote-atom, v1.1.9
scope-inspector, v0.2.1
script, v2.15.1

# Dev
No dev packages
```

/cc @atom/core

**Comment by GabLeRoux** [comment]
Had same Exception

For some reason, I had a right panel saying `GLOBAL` with a list with one element marked as `(anonymous function)    ()`. Clicked on that, thrown exception. On left panel, I had a `gruntfile.js` file open. 

I also experienced a strange behaviour with atom panels stuck on mouse position (constantly resizing panels), not quite sure this is due to this thought.

**Atom Version**: 0.189.0
**System**: Mac OS X 10.10.2
**Thrown From**: [scope-inspector](https://github.com/tvooo/scope-inspector) package, v0.2.1

### Stack Trace

Uncaught TypeError: Cannot read property 'editor' of null

```
At /Users/gableroux/.atom/packages/scope-inspector/lib/scope-inspector-view.coffee:60

TypeError: Cannot read property 'editor' of null
  at ScopeInspectorView.module.exports.ScopeInspectorView.onClickIdentifier (/Users/gableroux/.atom/packages/scope-inspector/lib/scope-inspector-view.coffee:60:29)
  at HTMLLIElement.<anonymous> (/Users/gableroux/.atom/packages/scope-inspector/lib/scope-inspector-view.coffee:21:41)
  at HTMLLIElement.handler (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/space-pen-extensions.js:112:34)
  at HTMLDivElement.jQuery.event.dispatch (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at HTMLDivElement.elemData.handle (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)

```

### Commands

```
     -0:46.3 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:46.3 core:cancel (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "metrics",
      "timecop",
      "vim-mode",
      "symfony-snippets",
      "atom-compile-coffee"
    ],
    "themes": [
      "seti-ui",
      "atom-dark-syntax"
    ],
    "excludeVcsIgnoredPaths": false,
    "followSymlinks": true
  },
  "scope-inspector": {
    "userId": "556582ae1428d3582b32e239e8f7ecac13c07688"
  }
}
```

### Installed Packages

```coffee
# User
ScssBundle, v0.4.0
angularjs, v0.1.0
ascii-codes, v0.3.3
atom-beautify, vundefined
atom-color-highlight, v3.0.8
atom-django, v0.2.1
atom-html-preview, v0.1.4
atom-jinja2, v0.5.0
autocomplete-plus, v2.8.0
bezier-curve-editor, v0.6.6
build, v0.27.0
coffee-lint, v0.7.3
coffee-refactor, v0.6.2
color-picker, v1.5.0
css-snippets, v0.5.0
dash, v1.1.0
editorconfig, v0.3.3
git-log, v0.3.0
grunt-runner, v0.8.0
gulp-snippets, v0.6.1
js-refactor, v0.3.0
jsonlint, v1.0.2
language-apache, v1.2.0
language-blade, v0.12.0
language-erlang, v2.0.0
language-gitignore, v0.1.1
language-hosts, v0.1.1
language-ini, v1.9.0
language-nginx, v0.4.0
language-python-django-templates, v0.1.0
language-salt, v0.2.0
laravel, v0.4.2
laravel-facades, v1.0.0
less-autocompile, v1.0.0
linter, v0.12.0
linter-js-yaml, v1.0.0
linter-pylint, v0.2.1
linter-scss-lint, v0.0.14
linter-xmllint, v0.0.5
local-history, v2.3.0
minimap, v4.7.5
minimap-git-diff, v4.1.2
minimap-highlight-selected, v4.2.0
php-twig, v3.0.0
pretty-json, v0.3.2
refactor, v0.4.1
resize-indent, v0.2.1
scope-inspector, v0.2.1
seti-ui, v0.6.3
ssh-config, v0.10.0
synced-sidebar, v0.2.3
travis-ci-status, v0.13.0

# Dev
No dev packages
```

**Comment by tvooo** [comment]
Was fixed by 0586fa52042efda1aef0f0d795a071bd6ca4581e

**Comment by GabLeRoux** [comment]
Awesome :+1: :)
